### ISSUE TITLE
Virus & Token grabber

### ISSUE THREAD

**Issue by vam3s** [SkAr7/DISCORD-IMAGE-TOKEN-LOGGER#1]
Its a "Output.exe" that grabs all your tokens and it also mines your PC.

**Comment by ManfaceClan** [comment]
yep it is lololol i tested on a alt and a vm and i had 2fa and it said somene tried to login into your account from dallas tx its just a vpn LOLOL

> Its a "Output.exe" that grabs all your tokens and it also mines your PC.
